tests/Admin/Admin_Page_Tests.php

```diff
@@ -118,4 +119,33 @@ public function test_render_page_with_no_plugins() {
 		$this->assertStringContainsString( 'No plugins available.', $output );
 		$this->assertStringNotContainsString( '<select id="plugin-check__plugins"', $output );
 	}
+
+	public function test_filter_plugin_action_links() {
+
+		$base_file = plugin_basename( WP_PLUGIN_CHECK_MAIN_FILE );
+
+		/** Editor check */
+		$editor = self::factory()->user->create( array( 'role' => 'editor' ) );
+		wp_set_current_user( $editor );
```

Comment:
No need to create an editor user here. If we just leave this out so no user is set, the `activate_plugins` capability is not granted either, so that works and makes the test a bit faster.

Reply:
Done